### DISCUSSION TITLE
[FEATURE-REQUEST] Chips/Tags en barra de canales

### DISCUSSION THREAD

**Discussion by Yiakman** [Cotalker/documentation#163] [3. Ideas & Feature Requests] [answered]
Muchas veces queremos categorizar conversaciones o tareas más allá de los estados propios de la implementación, por ejemplo para el caso de proyectos poner el área del cliente (agroindustria, minería, retail, etc). Esto ya es posible en tasks mediante los status1...5. Quizás sea un aporte extender alguno de estos marcadores a la barra de canales.

También se podrían diferenciar por colores de forma automática o seteada, ayudando a la distinción visual de los canales.

**Comment by acedward** [comment, marked as answer]
Está en el backlog, mejorar los marcadores y unificar la vista tareas y canales.
![image](https://user-images.githubusercontent.com/3951895/117719342-47611200-b1ab-11eb-8827-30b129c6231d.png)

Si necesitas priorizar esta funcionalidad, por favor contacta tu representante Cotalker
